src/app/page.tsx

```
@@ -0,0 +1,51 @@
+          <Link
+            className="btn btn-primary md:btn-lg btn-wide"
+            href="/builds/new"
+          >
+            Shape Your Build
+            <MoveRight size="1.2em" />
+          </Link>
+        </div>
+      </div>
+    </div>
+    <script
+      dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)